### PR TITLE
Fixed the encording issue for proworkflow,simplenote,supportbee and Basecamp

### PR DESCRIPTION
**Pull request by katheesR**

### CODE REVIEW

File: basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-access/grantAccess.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
```
Comment (prabathabey) [inline basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-access/grantAccess.xml]: License header is missing.

File: basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-attachments/createAttachment.xml [hunk 2]
```diff
@@ -12,7 +12,7 @@
```
Comment (prabathabey) [inline basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-attachments/createAttachment.xml]: License header is missing.

File: basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-calendarEvents/createCalendarEvent.xml [hunk 3]
```diff
@@ -88,7 +88,7 @@
```
Comment (prabathabey) [inline basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-calendarEvents/createCalendarEvent.xml]: License header is missing.

File: basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-calendarEvents/getCalendarEvent.xml [hunk 4]
```diff
@@ -21,7 +21,7 @@
```
Comment (prabathabey) [inline basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-calendarEvents/getCalendarEvent.xml]: License header is missing.

File: basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-access/grantAccess.xml (hunk 1 above)
Comment (katheesR) [inline basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-access/grantAccess.xml]: Fixed in the PR - #340

File: basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-attachments/createAttachment.xml (hunk 2 above)
Comment (katheesR) [inline basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-attachments/createAttachment.xml]: Fixed in the PR - #340

File: basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-calendarEvents/createCalendarEvent.xml (hunk 3 above)
Comment (katheesR) [inline basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-calendarEvents/createCalendarEvent.xml]: Fixed in the PR - #340

File: basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-calendarEvents/getCalendarEvent.xml (hunk 4 above)
Comment (katheesR) [inline basecamp/basecamp-connector/basecamp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/basecamp-calendarEvents/getCalendarEvent.xml]: Fixed in the PR - #340

**Pull request merged**